packages/schematics/package-update/_angular_cli/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { Rule } from '@angular-devkit/schematics';
+import { SchematicsUpdateSchema } from '../schema';
+import { updatePackageJson } from '../utility/npm';
+
+
+export default function(options: SchematicsUpdateSchema): Rule {
+  return updatePackageJson(['@angular/cli'], options.version, options.loose);
```

Comment:
Should updating Angular CLI itself should also update installed related tooling? Like Protractor and Karma versions.

Reply:
Those should be separate packages.